core/App/ehsm_napi.cpp

```diff
@@ -1326,6 +1326,7 @@ char *NAPI_RA_GET_API_KEY(const char *p_att_result_msg)
 
     char p_appid[UUID_STR_LEN] = {0};
```

Comment:
change the commit title as
"
transfer the apikey with ciphertext to the enroll app
"